test/integration/kic_custom_network_test.go

```diff
@@ -107,3 +128,15 @@ func verifyNetworkExists(ctx context.Context, t *testing.T, networkName string)
 		t.Fatalf("%s network is not listed by [%v]: %v", networkName, c.Args, output)
 	}
 }
+
+func verifySubnet(ctx context.Context, t *testing.T, network string, subnet string) {
```

Comment:
Omit repeated param (see comment above for more detail).

Reply:
Fixed.